packages/blockchain-wallet-v4-frontend/src/modals/SendBitcoin/FirstStep/template.success.js

```diff
@@ -148,33 +159,42 @@ const FirstStep = props => {
               <FormattedMessage id='modals.sendbitcoin.firststep.fee' defaultMessage='Transaction fee:' />
-            </Text>
-            {feeEditToggled
-              ? <FeeContainer>
-                <RowFlexEnd>
-                  <Link weight={300} size={'12px'} onClick={() => props.customFeeHandler(regular)}>Reg: {regular}</Link>
-                  <Link weight={300} size={'12px'} onClick={() => props.customFeeHandler(priority)}>Priority: {priority}</Link>
-                </RowFlexEnd>
-                {/* TODO: high and low limits for fee input */}
-                <Field name='fee' component={TextBox} validate={[required]} />
-                <Unit>sat/b</Unit>
-              </FeeContainer>
-              : <SelectFeeContainer>
-                <Field name='fee' component={SelectBoxFee} validate={[required]} />
-              </SelectFeeContainer>
-            }
-          </div>
-          {feeEditToggled
-            ? ''
-            : <Text size={'12px'} weight={300}>
-              {renderFeeConfirmationTime()}
-            </Text>}
+              {feeEditToggled
+                ? <FeeContainer>
+                  <RowFlexEnd>
+                    <Link weight={300} size={'12px'} onClick={() => props.customFeeHandler(regular)}>Reg: {regular}</Link>
+                    <Link weight={300} size={'12px'} onClick={() => props.customFeeHandler(priority)}>Priority: {priority}</Link>
+                  </RowFlexEnd>
+                </FeeContainer>
+                : <Field name='fee' inline component={NativeSelect} validate={[required]}>
+                  <option value={regular}>Regular</option>
+                  <option value={priority}>Priority</option>
+                </Field>
+              }
+            </FeeFormLabel>
+          </FeeFormContainer>
+          {
+            feeEditToggled && <FeeField name='fee' component={NumberBox} validate={[required]} />
+          }
+          {
+            !feeEditToggled && <Text size={'12px'} weight={300}>{renderFeeConfirmationTime()}</Text>
+          }
+          {
+            feeEditToggled && fee < limits.min && <FeeError size={'12px'} weight={300} color={'error'} onClick={() => props.customFeeHandler(limits.min)}>
+              <FormattedMessage id='modals.sendbitcoin.firststep.feebelowmin' defaultMessage='{min} sat/b or more is recommended' values={{min: limits.min}} />
+            </FeeError>
+          }
+          {
+            feeEditToggled && fee > limits.max && <FeeError size={'12px'} weight={300} color={'error'} onClick={() => props.customFeeHandler(limits.max)}>
+              <FormattedMessage id='modals.sendbitcoin.firststep.feeabovemax' defaultMessage='{max} sat/b or less is recommended' values={{max: limits.max}} />
+            </FeeError>
+          }
         </ColLeft>
         <ColRight>
-          <ComboDisplay coin='BTC'>{totalFee}</ComboDisplay>
+          <ComboDisplay size={'14px'} coin='BTC'>{totalFee}</ComboDisplay>
```

Comment:
This is pretty nitpicky, but I think we can pass the `size` prop as a string. Not sure it needs braces.

Reply:
Made the changes and removed other places in the code we use size={string} to be just size=string. Thanks